src/ml_tooling/baseclass.py

```diff
@@ -377,21 +384,35 @@ def gridsearch(self,
             ResultGroup object containing each individual score
         """
 
+        baseline_model = clone(self.model)
+        train_x, train_y = indexable(self.data.train_x, self.data.train_y)
```

Comment:
Since we are generating the data object ourselves, do we need this call? Alternatively, move it to the Data object?

Reply:
Moving in into Data. It also have the benifit of checking if data shapes match.